File: from-2201-to-2300/2275-largest-combination-with-bitwise-and-greater-than-zero/solution.py
class Solution:
    def largestCombination(self, candidates: List[int]) -> int:
        result = 0
        bitmask = 1
        for i in range(32):
            count = 0
            for n in candidates:
                count += 1 if bitmask & n else 0
            result = max(result, count)
            bitmask <<= 1
        return result

File: from-2201-to-2300/2275-largest-combination-with-bitwise-and-greater-than-zero/solution.go
func largestCombination(candidates []int) int {
    result := 0
    bitmask := 1
    for i := 0; i < 32; i ++ {
        count := 0
        for _, n := range(candidates) {
            if bitmask & n != 0 {
                count += 1
            }
        }
        if count > result {
            result = count
        }
        bitmask <<= 1
    }
    return result
}